website/views/shared/profiles/stats/costume.jade

```diff
@@ -0,0 +1,6 @@
+div(ng-show='profile.preferences.costume')
```

Comment:
`ng-if` would be better.  That way, if the user doesn't use a costume, this entire table doesn't have to be generated whenever the page loads.